src/reason_pprint_ast.ml

```diff
@@ -697,6 +697,9 @@ let rules = [
   [
     (TokenPrecedence, (fun s -> (Nonassoc, s = "=>")));
   ];
+  [
+    (TokenPrecedence, (fun s -> (Left, s.[0] == '~')));
```

Comment:
The order in this list should correspond to the ranking in precedence from `reason_parser.mly`. Notice how it corresponds for things like `&&` and `||`. The new infix `~` should similarly be represented.